tests/cases/conformance/es6/modules/exportBinding.ts

```diff
@@ -0,0 +1,5 @@
+export { x }
+const x = 'x'
+
+export { Y as Z }
```

Comment:
Strangely, this was not an error; only `const` and `let` were errors, even though `class Foo {}` has the exact same behavior as `let Foo = class {}` (no hoisting).